tensorflow/java/README.md

```diff
@@ -120,8 +120,8 @@ libraries will need to be built from source.
       //tensorflow/java:libtensorflow_jni
     ```
 
-The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.so`) will 
-be in `bazel-bin/tensorflow/java`.
+The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.dylib`) will 
```

Comment:
Oh, so it seems like you can use either/or. Is that correct?
https://www.chilkatsoft.com/java-loadLibrary-Linux.asp